src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Startup.cs

```diff
@@ -98,12 +99,12 @@ public void ConfigureServices(IServiceCollection services)
         // cause the data source builder to throw a host exception.
         if (!hostEnvironment.IsIntegrationTest())
         {
-            var connectionString = configuration.GetConnectionString("PublicDataDb")!;
+            var connectionString = ConnectionUtils.GetPostgreSqlConnectionString("PublicDataDb")!;
 
             if (hostEnvironment.IsDevelopment())
             {
                 var dataSourceBuilder = new NpgsqlDataSourceBuilder(
-                    configuration.GetConnectionString("PublicDataDb"));
+                    ConnectionUtils.GetPostgreSqlConnectionString("PublicDataDb"));
```

Comment:
I think this can be `var dataSourceBuilder = new NpgsqlDataSourceBuilder(connectionString);` because `connectionString` already exists a few lines above.

Reply:
Updated ta!